pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
I'm trying to install the required deps on Travis (CI environment), getting this error:

```
testBatchExtractionWithOCR(technology.tabula.TestBatchExtractor)  Time elapsed: 4.324 sec  <<< ERROR!

java.lang.RuntimeException: Unable to load library 'gs': Native library (linux-x86-64/libgs.so) not found in resource path ([file:/home/travis/build/tabulapdf/tabula-java/target/surefire/surefirebooter8727177790678118125.jar])

Please download, install GPL Ghostscript from http://sourceforge.net/projects/ghostscript/files

and/or set the appropriate environment variable.

	at net.sourceforge.tess4j.util.PdfUtilities.convertPdf2Tiff(PdfUtilities.java:55)

	at technology.tabula.extractors.OcrConverter.extract(OcrConverter.java:39)

	at technology.tabula.extractors.BatchSelectionExtractor.extract(BatchSelectionExtractor.java:214)

	at technology.tabula.TestBatchExtractor.testBatchExtractionWithOCR(TestBatchExtractor.java:70)

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

	at java.lang.reflect.Method.invoke(Method.java:498)

	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)

	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
...
```

What should we install on Ubuntu besides `tesseract-ocr`?